pom.xml

```diff
@@ -163,8 +163,9 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-context</artifactId>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <scope>runtime</scope>
```

Comment:
The <!-- Provided === -->, <!-- Compile === -->, etc comments are used for organizing dependencies by scope so someone who looks at the pom can see dependencies visually grouped by scope.  

So it'd be helpful to keep to that convention and put runtime dependencies together.